test/package.json

```
@@ -6,14 +6,14 @@
   "scripts": {
     "debug": "node --preserve-symlinks test.js test --framework karma --log-level debug",
     "test": "node --preserve-symlinks test.js test --framework karma",
-    "coverage": "node --preserve-symlinks test.js test --framework karma --config ./node_modules/@bedrock/karma/lib/coverageConfig.js",
-    "coverage-ci": "node --preserve-symlinks test.js test --framework karma --config ./node_modules/@bedrock/karma/lib/coverageConfig.js"
+    "coverage": "cross-env NODE_ENV=test c8 npm run test",
+    "coverage-ci": "cross-env NODE_ENV=test c8 --reporter=lcovonly --reporter=text-summary --reporter=text npm run test"
```

c8 is for node code, we have some test karma commands for coverage, though nothing happens in the tests here so they do nothing:
```suggestion
    "coverage": "node --preserve-symlinks test.js test --framework karma-coverage",
    "coverage-ci": "node --preserve-symlinks test.js test --framework karma-coverage-ci"
```